elixir/apps/web/lib/web/live/settings/identity_providers/jumpcloud/show.ex

```diff
@@ -244,7 +244,8 @@ defmodule Web.Settings.IdentityProviders.JumpCloud.Show do
   end
 
   def handle_event("delete", _params, socket) do
-    {:ok, _provider} = Auth.delete_provider(socket.assigns.provider, socket.assigns.subject)
+    {:ok, _deleted_provider} =
```

Comment:
Shouldn't have wrapped ?

Reply:
Turns out the added `_deleted` was just long enough to force this line to wrap